mqtt/mqtt-broker-tests-util/Cargo.toml

```diff
@@ -17,4 +17,5 @@ tracing = "0.1"
 tracing-subscriber = "0.1"
 
 mqtt3 = { path = "../mqtt3", features = ["serde1"] }
-mqtt-broker = { path = "../mqtt-broker" }
\ No newline at end of file
+mqtt-broker = { path = "../mqtt-broker" }
```

Comment:
It does seem like a circular dependency to me, but everything builds. I think it might have something to do with tests util being a dev dependency for the broker?